src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java

```diff
@@ -61,10 +61,14 @@
     public static final String CONFIG_DOCUMENTATION_MQ_USER_NAME = "The user name for authenticating with the queue manager.";
     public static final String CONFIG_DISPLAY_MQ_USER_NAME = "User name";
 
-    public static final String CONFIG_NAME_MQ_PASSWORD = "mq.password"; 
+    public static final String CONFIG_NAME_MQ_PASSWORD = "mq.password";
     public static final String CONFIG_DOCUMENTATION_MQ_PASSWORD = "The password for authenticating with the queue manager.";
     public static final String CONFIG_DISPLAY_MQ_PASSWORD = "Password";
 
+    public static final String CONFIG_NAME_MQ_CSP = "mq.csp";
+    public static final String CONFIG_DOCUMENTATION_MQ_CSP = "useMQCSPauthentication.";
+    public static final String CONFIG_DISPLAY_MQ_CSP = "useMQCSPauthentication";
```

Comment:
This is intended to be presented in a UI alongside the other options. This one doesn't follow the existing style.

Reply:
done